extensions/wikia/AdEngine/js/slot/service/srcProvider.js

```diff
@@ -0,0 +1,46 @@
+/*global define*/
+define('ext.wikia.adEngine.slot.service.srcProvider',  [
+	'ext.wikia.adEngine.adContext',
+	'ext.wikia.aRecoveryEngine.adBlockDetection',
+	'ext.wikia.aRecoveryEngine.adBlockRecovery',
+	require.optional('ext.wikia.aRecoveryEngine.instartLogic.recovery')
+], function (
+	adContext,
+	adBlockDetection,
+	adBlockRecovery,
+	instartLogic
+) {
+	'use strict';
+
+	function adIsRecoverable(extra) {
+		return extra && (extra.isPageFairRecoverable || extra.isInstartLogicRecoverable);
+	}
+
+	function isRecoverableByPF(extra) {
+		return adBlockRecovery.isEnabled() && adBlockDetection.isBlocking() && adIsRecoverable(extra);
+	}
+
+	function isRecoverableByIL() {
+		return instartLogic && instartLogic.isEnabled() && instartLogic.isBlocking();
+	}
+
+	function get(originalSrc, extra) {
+		console.log('---', adContext.get('opts.premiumOnly'));
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this logging statement. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarqube.wikia-inc.com/coding_rules#rule_key=javascript%3AS2228)